files/scripts/gbsyncd.sh

```diff
@@ -10,6 +10,7 @@ function startplatform() {
         if [ -z $($DB_CLI CONFIG_DB HGET 'FEATURE|gbsyncd' state) ]; then
             local CMD="local r=redis.call('DUMP', KEYS[1]); redis.call('RESTORE', KEYS[2], 0, r)"
             $DB_CLI CONFIG_DB EVAL "$CMD" 2 'FEATURE|syncd' 'FEATURE|gbsyncd'
+            $DB_CLI CONFIG_DB EVAL "$CMD" 2 'SYSLOG_CONFIG_FEATURE|syncd' 'SYSLOG_CONFIG_FEATURE|gbsyncd'
```

Comment:
i do not understand this, i search the buildimage, i do not see other container have this in their startup script.
how other containers handle this SYSLOG_CONFIG_FEATURE?

Reply:
Most of other containers are enabled by default which SONiC will create `SYSLOG_CONFIG_FEAGTURE|{container name}` in CONFIG_DB.

This PR is a kind of workaround inside container. But I think we might need a better approach to handle this kind of container from SONiC level.

Reply:
i understand now. "which is further because gbsyncd feature is for not enabled through init_cfg.json.j2."

Reply:
Why is there a separate `SYSLOG_CONFIG_FEATURE` object? Can we put syslog_config under `FEATURE` object?